app/assets/javascripts/mumuki_laboratory/application/faqs.js

```diff
@@ -0,0 +1,80 @@
+mumuki.faqs = class {
+
+  constructor() {
+    this.faqs = $('.mu-faqs');
+    this.topHierarchyElem = "H2";
+  }
+
+  // ================
+  // == Public API ==
+  // ================
+
+  /**
+   * Actually setup the faqs page
+   */
+  load() {
+    if(!this.faqs) return;
+    this._createFaqsGroups();
+    this._createNavbar();
+  }
+
+  // =================
+  // == Private API ==
+  // =================
+
+  _createNavbar() {
+    const $faqsNavbar = $(".mu-faqs-navbar nav ul");
+    $('.mu-faqs-group').each((_index, faqGroup) => {
+      const $navItem = this._createNavbarItem($faqsNavbar, faqGroup)
+      const $faqGroup = $(faqGroup);
+      this._configureClickFor($navItem, $faqGroup, $faqsNavbar)
+    });
+  }
+
+  _createNavbarItem($faqsNavbar, faqGroup) {
+    const $newLink = $('<a></a>').attr("href",`#${faqGroup.id}`)
+      .html(`${faqGroup.children[0].textContent}`);
+    const $navItem = $('<li></li>').html($newLink);
```

Comment:
I think you could do this two lines in a single like <code>$(`<li><a href=......`)</code> 

This is usually easier to read I think

Reply:
Also you could do something like this
```javascript
$('<li>', {
  html: $('<a>', { href: 'hola' }) 
})
```
But maybe is an overhead but consistent with what you're doing. The simple way is like @flbulgarelli suggest